nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -94,50 +96,46 @@ def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
         return blob
 
     @staticmethod
-    def get_bias_value(model: onnx.ModelProto, node: NNCFNode) -> np.ndarray:
+    def get_bias_value(model: onnx.ModelProto, bias_node: NNCFNode) -> np.ndarray:
         onnx_graph = ONNXGraph(model)
-        onnx_node = onnx_graph.get_node_by_name(node.node_name)
+        onnx_node = onnx_graph.get_node_by_name(bias_node.node_name)
         bias_port_id = onnx_graph.get_bias_tensor_port_id(onnx_node)
         bias_input_name = onnx_node.input[bias_port_id]
         if onnx_graph.has_initializer(bias_input_name):
-            return onnx_graph.get_initializers_value(bias_input_name)
-        node = onnx_graph.get_nodes_by_output(bias_input_name)[0]
-        metatype = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name(node.op_type)
+            bias_value = onnx_graph.get_initializers_value(bias_input_name)
+            return bias_value, bias_value.shape
```

Comment:
Could you please explain why we need to return `shape` here?

Reply:
It's needed for reshaping the new bias value in the algorithm.